content/slice/language-guide/custom-types.md

```diff
@@ -54,8 +54,8 @@ public static class BigIntSliceEncoderExtensions
 ```
 
 {% slice1 %}
-If you use this custom type as an optional (with a `?` suffix) outside of a tag context, you also need to supply methods
-to encode/decode a nullable value of this type. With our example:
+If this custom type is ever tagged or used as an optional type (with a `?` suffix), you also need to supply methods to
```

Comment:
I find this clarification confusing. tagged implies optional. Why not a simpler:

if this customer type is even used as an optional (with a `?` suffix), you also need to supply methods...

i.e. just remove the "outside of a tag context".

Reply:
I'll remove it.
I kept the distinction to be on the safe side. Yes, they both use the optional syntax, but in some places we make a distinction between optional (but not tagged) and 'pure' optional types.
But I've no problem with your suggestion.